app/src/services/ats.ts

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios';
+import config from 'config';
+
+import type { AxiosInstance } from 'axios';
+import type { ATSUserSearchParameters } from '../types';
+
+/**
+ * @function getToken
+ * Gets Auth token using ATS credentials
+ * @returns
+ */
+
+// async function getToken() {
+//   const response = await axios({
+//     method: 'POST',
+//     url: config.get('server.ats.tokenUrl'),
+//     data: {
+//       grant_type: 'client_credentials',
+//       client_id: config.get('server.ats.clientId'),
+//       client_secret: config.get('server.ats.clientSecret')
+//     },
+//     headers: {
+//       'Content-type': 'application/x-www-form-urlencoded'
+//     },
+//     withCredentials: true
+//   });
+//   return response.data.access_token;
+// }
```

Comment:
Please remove commented code if it is no longer required.

Reply:
done